src/database/services/quiz-api-service.ts

```diff
@@ -0,0 +1,102 @@
+import { DocumentClient } from "aws-sdk/clients/dynamodb";
+import { QuestionnaireModel } from "../schemas/questionnaire/questionnaire";
+
+const tableName = process.env.QUESTIONNAIRE_TABLE;
```

Comment:
Looking at the elastic app functions I don't think the table name needs to be an env. Lets remove it as an env (as we name it in the serverless.ts anyway) and just add it here as a string